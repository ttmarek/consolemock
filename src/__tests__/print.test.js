import makeConsoleMock from '../consolemock';

describe('print(message, [message1, ..., messageN])', () => {
  test('calling .print incorrectly throws a helpful error message', () => {
    const mock = makeConsoleMock();
    const callPrintIncorrectly = () => {
      mock.print();
    };

    expect(callPrintIncorrectly).toThrowErrorMatchingSnapshot();
  });
});
