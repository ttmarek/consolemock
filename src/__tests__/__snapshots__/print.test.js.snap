// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print(message, [message1, ..., messageN]) calling .print incorrectly throws a helpful error message 1`] = `
"[consolemock]
You called .print without giving makeConsoleMock a native console object:
  import { makeConsoleMock } from 'consolemock';
  console = makeConsoleMock(console);"
`;
